fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -1287,6 +1287,17 @@ private void computeScanRangeAssignmentByScheduler(
             Reference<Long> backendIdRef = new Reference<Long>();
             TNetworkAddress execHostPort = SimpleScheduler.getHost(minLocation.backend_id,
                     scanRangeLocations.getLocations(), this.idToBackend, backendIdRef);
+            if (execHostPort.hostname != minLocation.server.hostname ||
```

Comment:
!execHostPort.hostname.equals(minLocation.server.hostname)

Reply:
Thanks for your review. I have modified it.